src/tests.c

```diff
@@ -2451,13 +2451,10 @@ void run_field_convert(void) {
     CHECK(secp256k1_memcmp_var(&fes2, &fes, sizeof(fes)) == 0);
 }
 
-int fe_secp256k1_memcmp_var(const secp256k1_fe *a, const secp256k1_fe *b) {
-    secp256k1_fe t = *b;
-#ifdef VERIFY
-    t.magnitude = a->magnitude;
-    t.normalized = a->normalized;
-#endif
-    return secp256k1_memcmp_var(a, &t, sizeof(secp256k1_fe));
+int fe_memcmp(const secp256k1_fe *a, const secp256k1_fe *b) {
+    /* Compare only the struct member that holds the limbs
+       (there may be others in VERIFY mode). */
```

Comment:
Nit: Clarify that only the limbs matter this comparison?

```suggestion
    /* Compare only the struct member that holds the limbs
       (there may be others in VERIFY mode, but this function
       should ignore them). */
```